Makefile

```diff
@@ -147,6 +155,10 @@ gocovmerge: ## Download gocovmerge if necessary
 gcov2lcov: ## Download gcov2lcov if necessary
 	$(call go-install-tool,$(GCOV2LCOV),github.com/jandelgado/gcov2lcov@v1.0.5)
 
+.PHONY: hadolint
+hadolint: $(BASE) $(HADOLINT_TOOL); $(info  running hadolint...) @ ## Run hadolint
```

Comment:
please, add target for shellcheck

Reply:
Actually, we don't have any shell scripts in the repo, so I dropped the shellcheck scan altogether